gateway-server/src/main/java/org/apache/knox/gateway/websockets/WebsocketLogMessages.java

```diff
@@ -56,4 +56,8 @@ void failedCreatingWebSocket(
       text = "Websocket connection to backend server {0} closed")
   void onConnectionClose(String backend);
 
+  @Message(level = MessageLevel.DEBUG,
+      text = "{0}")
+  void debugLog(String message);
+
```

Comment:
Why do we need this? There is a `logMessage` method already (see line 53).

Reply:
As per comment from @moresandeep 'logMessage' is used to log websocket data messages rather than generic debug log messages. So added a new method.

Reply:
It's ok to have this new method; if I were you I'd rename the `logMessage` to `logWebsocketMessage` to express what it is actually doing.